src/minor-scales.js

```diff
@@ -0,0 +1,256 @@
+                >
+                  Gm:
+                </button>
+              </div>
+              <div className="flex-text">
+                <div>G A Bb C D Eb F G</div>
+              </div>
+              <div className="flex-accidentals">
+                <div>2 Flats</div>
+              </div>
+            </div>
+
+            <div className="flex-row">
+              <div className="flex-left">
+                <button
+                  className="note-button"
+                  onClick={() => soundPlay(audioClipsMinor[7])}
+                >
+                  C#m:
+                </button>
+              </div>
+              <div className="flex-text">
+                <div>C# D# E F# G# A B C#</div>
+              </div>
+              <div className="flex-accidentals">
+                <div>5 Sharps</div>
+              </div>
+            </div>
+
+            <div className="flex-row">
+              <div className="flex-left">
+                <button
+                  className="note-button"
+                  onClick={() => soundPlay(audioClipsMinor[8])}
+                >
+                  D#m:
+                </button>
+              </div>
+              <div className="flex-text">
+                <div>D# F F# G# A# B C# D#</div>
+              </div>
+              <div className="flex-accidentals">
+                <div>6 Sharps</div>
+              </div>
+            </div>
+
+            <div className="flex-row">
+              <div className="flex-left">
+                <button
+                  className="note-button"
+                  onClick={() => soundPlay(audioClipsMinor[9])}
+                >
+                  F#m:
+                </button>
+              </div>
+              <div className="flex-text">
+                <div>F# G# A B C# D E F#</div>
+              </div>
+              <div className="flex-accidentals">
+                <div>4 Sharps</div>
+              </div>
+            </div>
+
+            <div className="flex-row">
+              <div className="flex-left">
+                <button
+                  className="note-button"
+                  onClick={() => soundPlay(audioClipsMinor[10])}
+                >
+                  G#m:
+                </button>
+              </div>
+              <div className="flex-text">
+                <div>G# A# B C# D# E F# G#</div>
+              </div>
+              <div className="flex-accidentals">
+                <div>6 Sharps</div>
+              </div>
+            </div>
+
+            <div className="flex-row">
+              <div className="flex-left">
+                <button
+                  className="note-button"
+                  onClick={() => soundPlay(audioClipsMinor[11])}
+                >
+                  Bb:
+                </button>
+              </div>
+              <div className="flex-text">
+                <div>Bb C Db Eb F Gb Ab Bb</div>
```

Comment:
Same as with major scales, use functions to generate the HTML. See it like this, we try to let our computers work for us, not the other way around. When you write repetitive code, ask yourself if you could create this code in an easier and dynamic way. If we don't strive for that, it would be like endlessly filling in excel tables by hand.